packages/video_player/lib/video_player.dart

```diff
@@ -148,14 +150,15 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   /// package and null otherwise.
   VideoPlayerController.asset(this.dataSource, {this.package})
       : dataSourceType = DataSourceType.asset,
+        formatHint = null,
         super(VideoPlayerValue(duration: null));
 
   /// Constructs a [VideoPlayerController] playing a video from obtained from
   /// the network.
   ///
   /// The URI for the video is given by the [dataSource] argument and must not be
   /// null.
-  VideoPlayerController.network(this.dataSource)
+  VideoPlayerController.network(this.dataSource, {this.formatHint})
```

Comment:
Add doc to describe formatHint. Also indicate it only works for android

Reply:
I did have (Android only), but I've moved to a new line and added `Android only` to the start.